new/PRS13947Eleskndros.xml

```diff
@@ -47,15 +47,16 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 <person sex="1">
                     <persName xml:lang="gez" xml:id="n1">እሌስክንድሮስ</persName>
                     <persName xml:lang="gez" type="normalized" corresp="#n1">ʾƎleskǝndros</persName>
+                    <persName xml:lang="en" corresp="#n1" type="main">Alexander</persName>
                     <persName xml:lang="gez" xml:id="n2">እለ፡ እስክንድሮስ</persName>
                     <persName xml:lang="gez" type="normalized" corresp="#n2">ʾƎla ʾǝskǝndros</persName>
-                  <persName xml:lang="en" corresp="#n1" sameAs="#n2">Alexander</persName>
+                    <persName xml:lang="en" corresp="#n2" type="main">Alexander</persName>
```

Comment:
only one (`n1`) may be type="main", this is what the app must display, please remove the entire translation or at least type="main" here

Reply:
So, I will delete  the entire translation for n2. Is that correct?

Reply:
yes, that would be better